packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/ImageAndTextViewer/ImageAndTextViewer.spec.js

```diff
@@ -0,0 +1,197 @@
+import { expect } from 'chai'
+import { Grommet } from 'grommet'
+import { Provider } from 'mobx-react'
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import asyncStates from '@zooniverse/async-states'
+import zooTheme from '@zooniverse/grommet-theme'
+
+import { SubjectFactory } from '@test/factories'
+import mockStore from '@test/mockStore'
+
+import ImageAndTextViewerConnector from './ImageAndTextViewerConnector'
+
+describe('ImageAndTextViewer', function () {
+  function withStore (store) {
+    return function Wrapper ({ children }) {
+      return (
+        <Grommet theme={zooTheme}>
+          <Provider classifierStore={store}>
+            {children}
+          </Provider>
+        </Grommet>
+      )
+    }
+  }
+
+  const imageAndTextSubjectSnapshot = SubjectFactory.build({
+    content: 'This is test subject content.',
+    contentLoadingState: asyncStates.success,
+    locations: [
+      {
+        'image/jpeg': 'https://some.domain/image.jpg'
+      },
+      {
+        'text/plain': 'https://some.domain/text.txt'
+      }
+    ]
+  })
+
+  const store = mockStore({
+    subject: imageAndTextSubjectSnapshot
+  })
+
+  describe('with loading state of error', function () {
+    it('should render something went wrong', function () {
+      render(
+        <ImageAndTextViewerConnector
+          loadingState={asyncStates.error}
+          subject={imageAndTextSubjectSnapshot}
```

Comment:
```suggestion
          subject={store.subjects.active}
```
The subject viewer passes the active subject down, so I'd try using that in tests. 

The snapshot will pass the tests here, but the prop that the viewer expects is a Subject, not a snapshot.